### DISCUSSION TITLE
Difference between Java version vs non java version of EDCT

### DISCUSSION THREAD

**Discussion by Falcon911** [Fi0x/EDCT#114] [Q&A] [answered]
Please excuse the ingnorance on this question. But, is there a difference  between the JRE version vs the non-jre version? Maybe installing java onj my PC is better???

**Comment by Fi0x** [comment, marked as answer]
It's easiest if you use the `edctsetup.exe`, since it will handle all the java-stuff.
If you however already have  a java version installed, you can also use the `EDCT.exe`. But if your version is of java is too old, it won't work.
